<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>table</class>
 <widget class="QDialog" name="table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>976</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="table_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>8</number>
   </property>
   <property name="columnCount">
    <number>8</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="table_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>8</number>
   </property>
   <property name="columnCount">
    <number>8</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="table_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>8</number>
   </property>
   <property name="columnCount">
    <number>8</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="table_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>8</number>
   </property>
   <property name="columnCount">
    <number>8</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="DataDisplay" name="mainWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <widget class="DataDisplayText" name="mainWidget_4" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>401</width>
         <height>251</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>3</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="Gauss" name="mainWidget_3" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>458</width>
         <height>284</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="DataDisplay" name="mainWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>3</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <widget class="DataDisplayText" name="mainWidget_6" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>251</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
    <widget class="Gauss" name="mainWidget_5" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>458</width>
       <height>284</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="showImage">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>VL1 Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>VL2 Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>580</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>VL1 Error Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>580</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>VL2 Error Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>70</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Gauss</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="tableChart" name="chart" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>660</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="tableChart" name="chart_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>940</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>VL2 Mean Values</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>40</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Values</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>400</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Series</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>470</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="errorMetricComboBox">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>500</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Maximum Error</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mean Squared Error</string>
    </property>
   </item>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>680</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>940</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Y Axe Scale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveMeasurementSeriesButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>530</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sumErrorSensor1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sumErrorSensor2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>600</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Sum of Errors</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>630</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Sum of Errors</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minValueSensor1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>600</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minValueSensor2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>600</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>630</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Closest Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>630</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Closest Distance</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>560</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="CameraWidget" name="cameraWidget" native="true">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="CameraWidget" name="cameraWidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>19</y>
     <width>402</width>
     <height>252</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>940</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>VL1 Mean Values</string>
   </property>
  </widget>
  <widget class="QSlider" name="transparency">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>300</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>120</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="zoomInButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOutButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="panUpButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="panDownButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="panRightButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>25</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="panLeftButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>210</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="zoomInButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOutButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="panUpButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="panDownButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="panRightButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>25</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="panLeftButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>190</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Right camera</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>280</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Transparency</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>100</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Left camera</string>
   </property>
  </widget>
  <zorder>table_1</zorder>
  <zorder>table_2</zorder>
  <zorder>table_3</zorder>
  <zorder>table_4</zorder>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
  <zorder>showImage</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButton</zorder>
  <zorder>frame_5</zorder>
  <zorder>frame_6</zorder>
  <zorder>label_6</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>spinBox</zorder>
  <zorder>errorMetricComboBox</zorder>
  <zorder>verticalSlider</zorder>
  <zorder>label_8</zorder>
  <zorder>saveMeasurementSeriesButton</zorder>
  <zorder>sumErrorSensor1</zorder>
  <zorder>sumErrorSensor2</zorder>
  <zorder>label_7</zorder>
  <zorder>label_9</zorder>
  <zorder>minValueSensor1</zorder>
  <zorder>minValueSensor2</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>progressBar</zorder>
  <zorder>label_12</zorder>
  <zorder>transparency</zorder>
  <zorder>cameraWidget</zorder>
  <zorder>cameraWidget_2</zorder>
  <zorder>frame_3</zorder>
  <zorder>frame_4</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataDisplay</class>
   <extends>QWidget</extends>
   <header>datadisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tableChart</class>
   <extends>QWidget</extends>
   <header>tablechart.h</header>
   <container>1</container>
   <slots>
    <slot>updateYAxisMax(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>DataDisplayText</class>
   <extends>QWidget</extends>
   <header>datadisplaytext.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Gauss</class>
   <extends>QWidget</extends>
   <header>gauss.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CameraWidget</class>
   <extends>QWidget</extends>
   <header>camerawidget.h</header>
   <container>1</container>
   <slots>
    <slot>toggleVisibility()</slot>
    <slot>setTransparency(int)</slot>
    <slot>onTransparencyChanged(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>table</receiver>
   <slot>ValuesClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>951</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>998</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>table</receiver>
   <slot>startMeasurementSeries()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>957</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>991</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>table</receiver>
   <slot>setCustomMeasurementCount(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>955</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>977</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chart_2</receiver>
   <slot>updateYAxisMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>908</x>
     <y>870</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>778</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chart</receiver>
   <slot>updateYAxisMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>773</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>713</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveMeasurementSeriesButton</sender>
   <signal>clicked()</signal>
   <receiver>table</receiver>
   <slot>saveMeasurementSeries()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>952</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>965</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>table</receiver>
   <slot>GaussClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>953</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>935</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showImage</sender>
   <signal>clicked()</signal>
   <receiver>table</receiver>
   <slot>cameraClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>968</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>979</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transparency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cameraWidget</receiver>
   <slot>onTransparencyChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>969</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transparency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cameraWidget_2</receiver>
   <slot>onTransparencyChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>948</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>GaussClicked()</slot>
  <slot>ValuesClicked()</slot>
  <slot>startMeasurementSeries()</slot>
  <slot>setCustomMeasurementCount(int)</slot>
  <slot>updateYAxisMax(int)</slot>
  <slot>saveMeasurementSeries()</slot>
  <slot>cameraClicked()</slot>
 </slots>
</ui>
